src/components/svg/keys/svg-keyboard-key/svg-keyboard-key.component.ts

```diff
@@ -68,7 +69,11 @@ export class SvgKeyboardKeyComponent implements OnInit, OnChanges, OnDestroy {
     private subscription: Subscription;
     private animation: string = 'inactive';
 
-    constructor(private mapper: MapperService, private store: Store<AppState>) {
+    @HostListener('click') onClick() {
```

Comment:
Why do we need this? The parent could get the nativeElement with using a local template variable.